API.md

```diff
@@ -362,3 +362,43 @@ angular.module('yourApp', ['validation'])
 		};
     }]);
 ```
+
+
+### **Setup Message Element in config phase**
```

Comment:
Could you explain more in plain English? For example, "You could use this addMsgElement to ... It reduces your effort of manually putting `message-id" so on and so forth.